payloads/v9/_interactions/_applicationCommands/chatInput.ts

```diff
@@ -54,6 +55,17 @@ export interface APIApplicationCommandArgumentOptions extends Omit<APIApplicatio
 	choices?: APIApplicationCommandOptionChoice[];
 }
 
+/**
+ * This type is exported as a way to make it stricter for you when you're writing your commands
+ *
+ * In contrast to `APIApplicationCommandSubCommandOptions` and `APIApplicationCommandArgumentOptions`,
+ *  these types cannot have an `options` array, or a `choices` array, but they can have a `channel_types` one.
+ */
+export interface APIApplicationCommandChannelOptions extends Omit<APIApplicationCommandOptionBase, 'type'> {
+	type: ApplicationCommandOptionType.Channel;
```

Comment:
Not related to this line but please remove line 16 in this file! :D

Reply:
Gotcha, removed it from the deno version as well.